exporter/opentelemetry-exporter-zipkin/src/opentelemetry/exporter/zipkin/__init__.py

```diff
@@ -64,25 +64,37 @@
 import json
 import logging
 import os
-from typing import Optional, Sequence
+from typing import Optional, Sequence, Union
 from urllib.parse import urlparse
 
 import requests
 
+from opentelemetry.exporter.zipkin import zipkin_pb2
 from opentelemetry.sdk.trace.export import SpanExporter, SpanExportResult
 from opentelemetry.trace import Span, SpanContext, SpanKind
 
+TRANSPORT_FORMAT_JSON = "application/json"
+TRANSPORT_FORMAT_PROTOBUF = "application/x-protobuf"
```

Comment:
Looks like we are overloading format identifier and HTTP context type headers into a single set of variables. I think these should be different so people configuring the exporter through code, config file or env var don't need to use the HTTP content type

Reply:
Had the same thought but forgot to circle back - will change.